src/Drupal/Driver/DrupalDriver.php

```diff
@@ -140,6 +140,14 @@ public function clearCache($type = NULL) {
   /**
    * {@inheritdoc}
    */
+  public function clearStaticCaches() {
```

Comment:
This class already has this method down near the bottom. The issue is that the method was added here without being added to the interface.